bin/lamassu-send-coins

```diff
@@ -80,7 +80,12 @@ settingsLoader.loadLatest()
                 console.log('Success.')
                 process.exit(0)
               })
-              .catch(console.log)
+              .catch(e => {
+                if (e.name === 'InsufficientFundsError')
+                  console.log(e.message)
+                else
+                  console.log(e)
+              })
```

Comment:
Shouldn't we always print the message only at all times? 

From the card: "Instead, if there's an error, it's enough to show only the first error line"

Reply:
Oh damn, I thought that we want only to do this for `InsufficientFundsError`.
Fixing right now